chrome/views/popupView.js

```diff
@@ -1167,28 +1284,6 @@ GmailToTrello.PopupView.prototype.reset = function() {
     this.$popupContent.show();
 };
 
-GmailToTrello.PopupView.prototype.displaySubmitCompleteForm = function() {
-    var self = this;
-    var data = this.data.newCard;
-    gtt_log('displaySubmitCompleteForm: ' + this.data);
-
-    // NB: this is a terrible hack. The existing showMessage displays HTML by directly substituting text strings.
-    // This is very dangerous (very succeptible to XSS attacks) and generally bad practice.  It should be either 
-    // switched to a templating system, or changed to use jQuery. For now, I've used this to fix
-    // vulnerabilities without having to completely rewrite the substitution part of this code.
-    // TODO(vijayp): clean this up in the future
-    var jQueryToRawHtml = function(jQueryObject) {
-        return jQueryObject.prop('outerHTML');
-    }
-    this.showMessage(self, '<a class="hideMsg" title="Dismiss message">&times;</a>Trello card updated: ' + 
-        jQueryToRawHtml($('<a>')
-            .attr('href', data.url)
-            .attr('target', '_blank')
-            .append(data.title))
-        );
-    this.$popupContent.hide();
-};
-
```

Comment:
I got rid of this because it was kind of the easiest way to handle, and I think the UI is fine. Before, it would display a confirmation message after you submit. Now it needs to handle the newly created card, which means that the button and the popup both need to be re-generated with the new card included (since the button is now green and has some new text inside of it, and the popup now has a new card displayed at the top of it). So it was easier to just close the popup and re-create rather than trying to keep it open, display the message, and also make the changes to the button and popup.